<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conexión ZigBee</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Monitoreo de datos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tablaDatos">
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Pozo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dato</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Frecuencia de recepción de datos</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Modo de operación</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="selectorModo">
               <item>
                <property name="text">
                 <string>Dual</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>En línea</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLCDNumber" name="displayTiempoRestante">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
               <property name="value" stdset="0">
                <double>88888.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="selectorTiempo">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>1 minuto</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15 minutos</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30 minutos</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1 hora</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="botonIniciarPeticion">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/VentanaZigBee/icons/play.png</normaloff>:/VentanaZigBee/icons/play.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Estado de los sensores</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tablaSensores">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(44, 62, 80);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <row>
           <property name="text">
            <string>La1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>La2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>La3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>La4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>La5</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Dirección</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tipo</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>as</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>Presión</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>sdf</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>Flujo</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>sag</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>Flujo</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>fsda</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>Presión</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>asdf</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string>Nivel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
